tests/integ/test_kmeans.py

```diff
@@ -41,14 +41,27 @@ def test_kmeans(sagemaker_session):
                         k=10, sagemaker_session=sagemaker_session, base_job_name='test-kmeans')
 
         kmeans.init_method = 'random'
-        kmeans.max_iterators = 1
+        kmeans.max_iterations = 1
         kmeans.tol = 1
         kmeans.num_trials = 1
         kmeans.local_init_method = 'kmeans++'
         kmeans.half_life_time_size = 1
         kmeans.epochs = 1
         kmeans.center_factor = 1
 
+        assert kmeans.hyperparameters() == dict(
+            init_method=kmeans.init_method,
+            local_lloyd_max_iter=str(kmeans.max_iterations),
+            local_lloyd_tol=str(kmeans.tol),
+            local_lloyd_num_trials=str(kmeans.num_trials),
+            local_lloyd_init_method=kmeans.local_init_method,
+            half_life_time_size=str(kmeans.half_life_time_size),
+            epochs=str(kmeans.epochs),
+            extra_center_factor=str(kmeans.center_factor),
+            k=str(kmeans.k),
+            force_dense='True'
```

Comment:
nit: I like to add a comma at the end of lines like these to make the diff cleaner if someone adds something in the future